src/Logging/Logging.Console/Extensions/LogService.Console.Extensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace CodeMonkeys.Logging.Console
+{
+    public static partial class LogServiceExtensions
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)